roles/dnsmasq/tasks/main.yml

```diff
@@ -108,6 +108,17 @@
     - timeout:2
     - attempts:2
 
+- name: Remove search and nameserver options from resolvconf base
+  lineinfile:
+    dest: /etc/resolvconf/resolv.conf.d/base
+    state: absent
+    regexp: "^{{ item }}.*$"
+    backup: yes
+    follow: yes
+  with_items:
+    - search
+    - nameserver
```

Comment:
-1 . Please make sure the command `resolvconf -u` below will not resurrect them. The changes you suggest have to be  persisted and running state to be consistent with it as well.